docs/core/project-sdk/msbuild-props.md

```diff
@@ -433,6 +435,38 @@ By default, in a new .NET project, this property is set to `true`. If set to `fa
 </PropertyGroup>
 ```
 
+### EnablePreviewFeatures
```

Comment:
@terrajobst Could you review the changes to this MSBuild property reference doc?